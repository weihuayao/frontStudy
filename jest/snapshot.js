/*
 * @Descripttion: 
 * @version: 
 * @Author: weihua.yao
 * @Date: 2020-03-17 13:46:12
 * @LastEditTime: 2020-03-17 13:47:59
 */
export const generateConfig = () => {
  return {
    serve: 'http://localhost',
    port: 8000,
  }
}