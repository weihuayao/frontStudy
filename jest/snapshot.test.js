import { generateConfig } from './snapshot'

test('测试generateConfig 函数', () => {
  expect(generateConfig()).toMatchSnapshot();
})