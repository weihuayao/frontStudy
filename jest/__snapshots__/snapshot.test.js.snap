// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`测试generateConfig 函数 1`] = `
Object {
  "port": 8000,
  "serve": "http://localhost",
}
`;
